library/std/src/os/wasi/io/raw.rs

```diff
@@ -2,4 +2,20 @@
 
 #![unstable(feature = "wasi_ext", issue = "71213")]
 
+// NOTE: despite the fact that this module is unstable,
+// stable Rust had the capability to access the stable
+// re-exported items from os::fd::raw through this
+// unstable module.
+// In PR #95956 the stability checker was changed to check
+// all path segments of an item rather than just the last,
+// which caused the aforementioned stable usage to regress
+// (see issue #99502).
+// As a result, the items in os::fd::raw were given the
+// rustc_allowed_through_unstable_modules attribute.
+// If new items are added to os::fd::raw and you want
+// them to be available through this module,
+// be sure to add that attribute to them.
+// Conversely, if this module is stabilized,
+// you may want to remove those attributes
+// (assuming no other unstable modules are using them).
```

Comment:
```suggestion
```

I don't think we should be encouraging new allowed_through_unstable items, especially because the existence of these exceptions essentially tie our hands on stabilizing their module path as is. Is there any reason to keep this module unstable at this point other than momentum?

Might be worth removing this bit of comment, I'll leave that up to you though.

Reply:
I'll remove that part of the comment.

> Is there any reason to keep this module unstable at this point other than momentum?

I think that's a question for the people behind the wasi-ext feature. In the past when I've asked about stabilizing any part of this feature, the response has been that people are reluctant to stabilize anything so long as WASI itself is unstable.